Assets/Mirror/Core/NetworkClient_TimeInterpolation.cs

```diff
@@ -93,8 +93,10 @@ public static partial class NetworkClient
         // initialization called from Awake
         static void InitTimeInterpolation()
         {
-            // reset timeline & snapshots from last session (if any)
+            // reset timeline, localTimescale, bufferTimeMultiplier & snapshots from last session (if any)
             localTimeline = 0;
+            localTimescale = 1;
+            bufferTimeMultiplier = 2;
```

Comment:
good PR.
hardcoding "=2" is a bit risky though, since the default may change in the future.
perhaps we need:
- "configuredBufferTimeMultiplier"  (set by user)
- "runtimeBufferTimeMultiplier" (modified by code)
- and in InitTimeInterp, set runtime = configured ?

Reply:
That can be done in a separate PR where you add configurable options to this system.

Reply:
actually, we may not need to reset bufferTimeMultiplier at all.
it's safe to assume the user's network connection won't change between sessions.
whatever the optimal multiplier was, could remain

Reply:
I can make that change if that's what you want, but I'll put a comment with it as to why we're not resetting it.

Confirm that's what you want?

Reply:
> it's safe to assume the user's network connection won't change between sessions. whatever the optimal multiplier was, could remain

I'd say that's a safe assumption for the most part... but keep in mind on mobile the device may switch between WiFi and cellular. Of course, Mirror would disconnect from the server if that happened since the original connection would suddenly get cut. Just keep that in mind, but I'd say 99% of the time it's safe to assume.

Reply:
reverted the reset of bufferTimeMultiplier - marking resolved.